src/pbt/v2/client/airflow_client/mwaa_client.py

```diff
@@ -0,0 +1,155 @@
+import base64
+import re
+from abc import ABC
+import datetime
+
+import boto3
+
+from src.pbt.v2.client.airflow_client.airflow_rest_client import AirflowRestClient
+from src.pbt.v2.exceptions import DagNotAvailableException, DagFileDeletionFailedException, DagUploadFailedException
+from src.pbt.v2.project_models import DAG
+import requests
+import json
+
+from src.pbt.v2.utility import retry
+
+
+class MWAARestClient(AirflowRestClient, ABC):
+
+    def __init__(self, environment_name: str, region: str, access_key: str, secret_key: str):
+        self.environment_name = environment_name
+        self.s3_aws_client = boto3.client(
+            "s3",
+            region_name=region,
+            aws_access_key_id=access_key,
+            aws_secret_access_key=secret_key
+        )
+
+        self.mwaa_client = boto3.client("mwaa", region_name=region, aws_access_key_id=access_key,
+                                        aws_secret_access_key=secret_key)
+
+        self.environment = self.mwaa_client.get_environment(Name=environment_name)
+        self.dag_s3_path = self.environment['Environment']['DagS3Path']
+        self._source_bucket = self._extract_bucket_name_from_arn()
+
+    def create_secret(self, key: str, value: str) -> bool:
+        return True
```

Comment:
?

Reply:
As PMA is on Composer, we didn't need to implement these methods in MWAA client.